app/build.gradle

```diff
@@ -1,11 +1,11 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion '28.0.2'
+    compileSdkVersion 29
+    buildToolsVersion '29.0.2'
     defaultConfig {
-        minSdkVersion 9
-        targetSdkVersion 26
+        minSdkVersion 14
```

Comment:
Please reset minSdkVersion to 9, it is just a loss (although small), and we actually don't gain anything for it as compensation in terms of features.

Reply:
Alright, I'll revert this.

Reply:
it's done